app/utils.py

```diff
@@ -101,7 +101,7 @@ def standardize_response(
     """
     data = payload.get("data")
     errors = payload.get("errors")
-    pagination_details = payload.get("pagination_details")
+    pagination_details = payload.get("details")
```

Comment:
This might be kind of nit-picky but now that I'm looking at it I think this is going to bother me. We should change `pagination_details` to `details` here and on lines 132 & 133, since it's not always pagination details and that could be confusing in the future.